netlify.toml

```diff
@@ -1,157 +1,250 @@
+#   status = 302               # HTTP status code
+
+# ========================================
+# General ILSA Links
+# ========================================
+
+# Rules and Documentation
 [[redirects]]
   from = "/rules"
   to = "/documents/ILSA%20-%20Rules%20of%20Play%20-%201.1.pdf"
   status = 302
+  # Short link to official rules of play
 
 [[redirects]]
   from = "/howtoplay"
   to = "/documents/ILSA%20-%20Rules%20of%20Play%20-%201.1.pdf"
   status = 302
-  # now points to full rules, later to lightweight rules
+  # Alternate URL for rules - eventually may point to simplified version
 
+# Forms and Signups
 [[redirects]]
   from = "/volunteer"
   to = "https://forms.gle/vYRcQb3JhAdks5cN7"
   status = 302
+  # Volunteer signup Google Form
 
 [[redirects]]
-  from = "/l/banner-join"
-  to = "/join"
-  status = 302
-
-[[redirects]]
-  from = "/l/banner-donate"
-  to = "/donate"
+  from = "/report-points"
+  to = "https://forms.gle/w9HrzHT5M4pikzFZA"
   status = 302
+  # Form for reporting tournament points
 
+# Social and External Links
 [[redirects]]
   from = "/l/bio"
   to = "https://lnk.bio/illinoisshuffleboard"
   status = 302
-
-[[redirects]]
-  from = "/report-points"
-  to = "https://forms.gle/w9HrzHT5M4pikzFZA"
-  status = 302
+  # Linktree-style bio page with multiple links
 
 [[redirects]]
   from = "/l/discord"
   to = "https://discord.gg/XUJm3ctWEw"
   status = 302
+  # ILSA Discord server invite
 
-#  ####### national tournament links ####### 
-
+# Internal Navigation
 [[redirects]]
-  from = "/lottery"
-  to = "https://docs.google.com/spreadsheets/d/1Gi-otAAGUBaqjHzIIM6zJWq4iBHNgxsxeN9GZFikBsc/edit?gid=1237102943#gid=1237102943"
+  from = "/l/banner-join"
+  to = "/join"
   status = 302
-  # now points to 2025 lottery selection google sheet
+  # Trackable link for banner/ad campaigns
 
 [[redirects]]
-  from = "/lottery-faq"
-  to = "https://drive.google.com/file/d/17U6DvWxn6JiykiQfhXXgop9HHJBi-7I-/view?usp=sharing"
+  from = "/l/banner-donate"
+  to = "/donate"
   status = 302
-  # now points to 2025 pdf
+  # Trackable link for donation campaigns
 
-[[redirects]]
-  from = "/standings"
-  to = "https://docs.google.com/spreadsheets/d/1FAIppioHDaAQNwZwvhCqJrbab_t3Adn5Nyk0LPAoiD0/edit?usp=sharing"
-  status = 302
-  # now points to 2025 bracket
+# ========================================
+# National Tournament Links
+# ========================================
+# These links are updated annually for each national tournament
+# Remember to update the year in comments when changing URLs
 
+# Tournament Information
 [[redirects]]
   from = "/national"
   to = "https://lnk.bio/ILLINOISSHUFFLEBOARD"
   status = 302
+  # Main national tournament information hub
 
 [[redirects]]
   from = "/nationals"
   to = "/national"
   status = 302
-
-[[redirects]]
-  from = "/national-store"
-  to = "https://chicago-national-tournament-merchandise.square.site/"
-  status = 302
+  # Alternate spelling redirects to /national
```

Comment:
This is for the merch store and doesn't redirect to /national at all, but the store for that